cf-deployment.yml

```diff
@@ -338,7 +339,13 @@ addons:
           deployment: cf
           network: default
           domain: bosh
-
+      - domain: log-cache.service.cf.internal
+        targets:
+        - deployment: cf
```

Comment:
Key-ordering is inconsistent with the other bosh-dns entries.

(I admire the commitment to alphabetical though)